src/util/MultiError.cc

```diff
@@ -54,6 +54,22 @@ std::string MultiError::toOneLineString() const {
 
 int MultiError::firstErrorCode() const { return empty() ? ErrorCode::NONE : _errorVector.front().getCode(); }
 
+string MultiError::firstErrorStr() const {
+    ostringstream os;
+    if (!empty()) {
+        os << _errorVector.front();
+    }
+    return os.str();
+}
```

Comment:
I would rewrite and speed up tfis metod as:
```
string MultiError::firstErrorStr() const {
    if (empty()) return string();
    stringstream os;
    os << _errorVector.front();
    return os.str();
}
```
This optimization will instantiate/constrict the stream object on the stack only if the collection is not empty. In most cases, it's empty.